src/consensus/parsec.rs

```diff
@@ -232,7 +231,7 @@ impl ParsecMap {
         &mut self,
         participants: BTreeSet<PublicId>,
         vote_id: &FullId,
-    ) -> Option<DkgResult> {
+    ) -> Option<parsec::DkgResult> {
```

Comment:
will check if can get this removed.

Reply:
Yip let's not mock here at all if possible. Critical to not use parsec methods as well. We will remove parsec soon.

Reply:
Had a check with attempt to remove this.
However, it turned out to affects many mock_parsec functions, and testing logic as well.
I'd prefer get this done in a separate PR, which will be much cleaner by itself.
Or even just wait for the removal of the parsec, so all the mock_parsec stuff can be gone at once.

Reply:
I'm OK with dealing with this in a separate PR. Maybe just add a `TODO` with something like "remove this function because we no longer use parsec for DKG".